### PR TITLE
package refactor

### PR DESCRIPTION
**Pull request by magiconair**
```
command/agent/*                  -> agent/*
command/consul/*                 -> agent/consul/*
command/agent/command{,_test}.go -> command/agent{,_test}.go
command/base/command.go          -> command/base.go
command/base/*                   -> command/*
commands.go                      -> command/commands.go
```

`agent/consul` will be refactored separately

### CODE REVIEW

File: command/base.go [hunk 1]
```diff
@@ -239,7 +239,7 @@ func (c *Command) helpFlagsFor(f *flag.FlagSet) string {
 			return
 		}
 		if firstCommand {
-			printTitle(&out, "Command Options")
+			printTitle(&out, "BaseCommand Options")
```
Comment (magiconair) [inline command/base.go]: This should probably remain `Command Options`

File: main.go [hunk 2]
```diff
@@ -21,7 +22,7 @@ func main() {
 func realMain() int {
 	log.SetOutput(ioutil.Discard)
 
-	// Get the command line args. We shortcut "--version" and "-v" to
+	// Get the c line args. We shortcut "--version" and "-v" to
```
Comment (slackpad) [inline main.go]: Search and replace fail :-)

File: main.go [hunk 3]
```diff
@@ -37,17 +38,17 @@ func realMain() int {
 		}
 	}
 
-	// Filter out the configtest command from the help display
+	// Filter out the configtest c from the help display
```
Comment (slackpad) [inline main.go]: Same here.

**Comment by magiconair** [comment]
@slackpad I've simplified `main.go` so that we don't need the comments. Could you have a look whether I've missed something, please?

**Comment by slackpad** [comment]
LGTM

**Comment by magiconair** [comment]
Thx. Going to rebase and merge now

**Pull request merged**